goodkey/blocked.go

```diff
@@ -0,0 +1,70 @@
+package goodkey
+
+import (
+	"crypto"
+	"errors"
+	"io/ioutil"
+
+	"github.com/letsencrypt/boulder/core"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+// blockedKeys is a type that maintains a map of Base64 encoded SHA256 hashes of
+// DER encoded PKIX public keys that should be considered blocked. blockedKeys
+// are created by using loadBlockedKeysList.
+type blockedKeys struct {
+	blockedMap map[string]bool
+}
+
+// blocked checks if the given public key is considered administratively
+// blocked based on a Base64 encoded SHA256 hash of the DER encoded PKIX public
+// key.
```

Comment:
as `blockedKeys` isn't protected by a lock can you add a short comment indicating it's not safe to use this until after `loadBlockedKeysList` has returned.

Reply:
Good idea :+1:

Reply:
Fixed in 3339956